app/src/main/res/layout/item_detail_bill.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:car_view="http://schemas.android.com/tools"
+    >
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="com.framgia.fsalon.screen.billdetail.BillDetailViewModel"
+            />
+
+        <variable
+            name="item"
+            type="com.framgia.fsalon.data.model.BillItemResponse"/>
+
+        <variable
+            name="position"
+            type="int"/>
+
+    </data>
+
+    <android.support.v7.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/dp_8"
+        car_view:cardElevation="@dimen/dp1"
+        >
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center"
+            android:orientation="horizontal"
+            >
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:background="@color/colorPrimary"
+                android:gravity="center"
+                android:minWidth="@dimen/dp_20"
+                android:padding="@dimen/dp_2"
+                android:text="@{position + 1 + ``}"
+                android:textColor="@color/color_white"
+                android:textSize="@dimen/sp_20"/>
+
+            <LinearLayout
+                android:layout_width="0dp"
+                android:layout_height="match_parent"
+                android:layout_weight="1"
+                android:orientation="vertical"
+                android:paddingLeft="@dimen/dp_3"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingLeft` you should probably also define `paddingRight` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.